doc/developer_guide/coding-style.rst

```diff
@@ -155,6 +155,47 @@ retained.). For example::
     except KeyError as err:
         raise InternalError("Useful message here") from err
 
+
+Logging
+-------
+
+PSyclone supports the Python `logging` module, to help enable improved
+general messages throughout the code. By default the logging infrastructure
+is switched off, but can be enabled either through the PSyclone script
+options or programatically by::
+
+    import logging
+    logging.basicConfig(level=logging.LEVEL)
+
+where `logging.LEVEL` is the required level of those defined in the standard
+logging library.
+
+To use logging in a module, you can add::
+
+    import logging
+    logger = logging.getLogger(__name__)
+    logging.info("Example of a logging message.")
+
+at the top of a module to make the logger available everywhere in the file.
+This is standard practice for using logging in Python.
+
+Python has 5 levels of information as standard in logging, `debug`, `info`,
```

Comment:
s/Python/Logging/
s/as standard in logging,/:/

Reply:
Done